hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/webapp/AbstractRESTRequestInterceptor.java

```diff
@@ -86,4 +92,31 @@ public RESTRequestInterceptor getNextInterceptor() {
     return this.nextInterceptor;
   }
 
+  private void setupUser(String userName) {
+    try {
+      if(userName == null || userName.isEmpty()){
```

Comment:
space after if

Reply:
Thank you very much for your help reviewing the code, I will fix it.